requirements_mysql.txt

```diff
@@ -1,7 +1,7 @@
-mysqlclient==2.0.2 \
-    --hash=sha256:3739824f4b8f29a6a4e5fc965f933fdce7c9edc2454990c3e5c6e038779e5580 \
-    --hash=sha256:4861e33b9fad83dadf38feae77a14106365fc8c354ec4300fab5d8ccfbee4b4e \
-    --hash=sha256:4d91dde55230a512937aafc5981c6c11c0efad2c6b3d33e83287208c8bdab367 \
-    --hash=sha256:8944d9008c29abfa43e3c97551416ea23b2ccb017af85682b79f34fc26dc43ae \
-    --hash=sha256:8df057b08fc27d8f7106bfa997d0a21e2acef017f905f06d6fb0aa6a20d4d2b2 \
+mysqlclient==2.0.3 \
```

Comment:
could you reverse this hash and version change. Other than that this PR looks good to me.

Reply:
Can do, but it looks like it will get changed again the next time someone runs `requirements_txt_updater.sh`. If you mean "let's not change this now", makes sense to me, so I'll go ahead and revert it; if it's "we want to avoid 2.0.3", that might need a different approach?

Reply:
It's "let's not change this now".

We have a convention to run the updater, then trim out any changes that aren't absolutely required by the PR at hand.

(And then periodically / as needed, create a PR which just updates the hashes, but that's separate from regular PRs that change other stuff.)

Reply:
Thanks, that helps clarify procedure.

In that case, I'll check if there's anything needed for this PR that isn't in the file on `develop` branch, and if not, I'll re-merge develop into this PR and use that version of `requirements.txt` verbatim.